x/epochstorage/keeper/stake_storage.go

```diff
@@ -256,7 +256,7 @@ func (k Keeper) ModifyStakeEntryCurrent(ctx sdk.Context, chainID string, stakeEn
 		// do nothing and return to avoid panic.
 		utils.LavaFormatError("critical: ModifyStakeEntryCurrent with unknown chain", legacyerrors.ErrNotFound,
 			utils.LogAttr("chainID", chainID),
-			utils.LogAttr("stakeAddr", stakeEntry.Address),
+			utils.LogAttr("stakeAddr", stakeEntry.Operator),
```

Comment:
I return the operator address in errors because it's the address that operates the provider process, so it'll make more sense to a user seeing the error (vault only holds funds)